library/VTKExtensions/Rendering/vtkF3DRenderer.cxx

```diff
@@ -1338,6 +1339,22 @@ void vtkF3DRenderer::ShowEdge(bool show)
   }
 }
 
+//----------------------------------------------------------------------------
+void vtkF3DRenderer::SetUseOrthographicProjection(bool use)
+{
+  vtkCamera* camera = GetActiveCamera();
+  if (use)
```

Comment:
```suggestion
  vtkCamera* camera = GetActiveCamera();
  if (use == camera->GetParallelProjection())
    return;
  if (use)
```
can we shortcut the computation if we're already in the right mode? or is there some cases where they need to be done anyway?

Reply:
We can, but lets use an internal state instead.

Reply:
I will investigate using the internal state